service/service.go

```diff
@@ -197,7 +196,6 @@ func New(config Config) (*Service, error) {
 				PrometheusAddress:           config.Viper.GetString(config.Flag.Service.Prometheus.Address),
 				PrometheusBaseDomain:        config.Viper.GetString(config.Flag.Service.Prometheus.BaseDomain),
 				PrometheusCreatePVC:         config.Viper.GetBool(config.Flag.Service.Prometheus.Storage.CreatePVC),
-				PrometheusStorageSize:       config.Viper.GetString(config.Flag.Service.Prometheus.Storage.Size),
```

Comment:
You can remove it from the configmap https://github.com/giantswarm/prometheus-meta-operator/blob/144cc87bb8a47efec40d7ab3a575b0c1d3eacd9e/helm/prometheus-meta-operator/templates/configmap.yaml#L51 and from the config repo (apart for KVM installations)